pypeit/wavetilts.py

```diff
@@ -446,7 +446,7 @@ def model_arc_continuum(self, debug=False):
             # TODO: What to do with the following iter_continuum parameters?:
             #       sigthresh, sigrej, niter_cont, cont_samp, cont_frac_fwhm
             arc_continuum[:,i], arc_fitmask[:,i] \
-                    = arc.iter_continuum(self.arccen[:,i], inmask=np.invert(self.arccen_bpm[:,i]),
+                    = arc.iter_continuum(self.arccen[:,i], gpm=np.invert(self.arccen_bpm[:,i]),
```

Comment:
it is odd that `inmask` is renamed here but no further changes occur in the module.  Is that ok?

Reply:
Yeah, was trying to slowly migrate to specifying gpm wherever we specify masks.